clang/lib/CIR/CodeGen/CIRGenValue.h

```diff
@@ -316,6 +317,20 @@ class LValue {
     return VectorIdx;
   }
 
+  // extended vector elements.
+  Address getExtVectorAddress() const {
+    assert(isExtVectorElt());
+    return Address(getExtVectorPointer(), ElementType, getAlignment());
+  }
+  mlir::Value getExtVectorPointer() const {
+    assert(isExtVectorElt());
+    return V;
+  }
+  mlir::Attribute getExtVectorElts() const {
```

Comment:
From overall usage in this PR, looks like this could return an `ArrayAttr` instead.